deployments/kustomize/base/backups/README.md

```diff
@@ -0,0 +1,102 @@
+# Backups
+In order to restore backups you will need to utilize the sidecar, this needs to be deployed, and the pod needs to be run. See deployment of sidecar for more information.  An additional patch to include the volume mounts for backups and restores will also needed to be added to the deployment.
```

Comment:
Maybe like you did above for backups, it could be helpful to include the link for deployment of sidecar.

Reply:
i know i was thinking about this as i was reviewing my pr. :(. I'm hoping that someone takes a kick at the documentation, and improves, it from the point of never done these steps before.